wordcloud/wordcloud.py

```diff
@@ -576,13 +576,14 @@ def process_text(self, text):
 
         flags = (re.UNICODE if sys.version < '3' and type(text) is unicode  # noqa: F821
                  else 0)
-        pattern = r"\w[\w']*" if self.min_word_length <= 1 else r"\w[\w']+"
-        regexp = self.regexp if self.regexp is not None else pattern
+        if self.regexp is not None:
```

Comment:
Thanks for the PR!
This call to compile actually doesn't change anything, as the object is only used once, right?

Reply:
@amueller,

Really sorry for the long delay. Yes there's no change on the workflow. The ideia is to simplify.

All the tests have passed locally, do you think it's useful to include any more?

Reply:
No worries, I'm only very sporadically on this project these days. How does this change about the re simplify, the code gets longer, right?